manim/mobject/stats.py

```diff
@@ -0,0 +1,134 @@
+r"""Mobjects that have to do with statistics and statistical analysis.
+
+Examples
+--------
+.. manim:: ShowHistogram
+    :save_last_frame:
+
+    class ShowHistogram(GraphScene)
```

Comment:
```suggestion
    class ShowHistogram(GraphScene):
```

Reply:
I actually just noticed that myself, and had already fixed it.